src/code/audio/audio_heap.c

```diff
@@ -1042,8 +1042,8 @@ void AudioHeap_Init(void) {
             AudioHeap_AllocDmaMemoryZeroed(&gAudioContext.miscPool, gAudioContext.maxAudioCmds * sizeof(u64));
     }
 
-    // Initialize unk_3520 (fadeOutVelocities for ADSR)
-    gAudioContext.unk_3520 = AudioHeap_Alloc(&gAudioContext.miscPool, 0x100 * sizeof(f32));
+    // Initialize adsrDecayTable (fadeOutVelocities for ADSR)
+    gAudioContext.adsrDecayTable = AudioHeap_Alloc(&gAudioContext.miscPool, 0x100 * sizeof(f32));
```

Comment:
These aren't static values actually, they can change on heap reset.

The audio thread runs at 60 fps, but the sequences/notes are processed multiple times for each of those frames. This is captured in the variable `updatesPerFrame` (`gAudioContext.audioBufferParameters.updatesPerFrame`). When I tested it in OoT, this tends to be either 3 or 4 i.e. the sequence will be processed 3 * 60 fps or 4 * 60 fps.

Now however, `updatesPerFrame` can change depending on the audio specifications Id you give the heap during reset. I believe it'll work out the same most of the time, but there is one audio specification that runs at 22.5 kHz instead of the more common 32 kHz, and that will change `updatesPerFrame`.

The `adsrDecayTable` is then scaled by `updatesPerFrame` so that it decrements at the same rate if `updatesPerFrame` changes